tests/conformance/conformance.go

```diff
@@ -9,27 +9,26 @@ import (
 
 var containerTag = ""
 
-func done(files []string) {
-	err := os.WriteFile("/tmp/sonobuoy/results/done", []byte(strings.Join(files, "\n")), 0666)
-	if err != nil {
-		fmt.Printf("Failed to notify sonobuoy that I am done: %v\n", err)
-	}
-}
+const resultsDir = "/tmp/sonobuoy/results"
 
 func main() {
 	err := execute()
-	done([]string{
-		"/tmp/sonobuoy/results",
-	})
+	if err != nil {
+		os.Exit(1)
```

Comment:
Print error message here too.

Reply:
error is printed already in the execute function?

Reply:
Ah, sorry, with too I referred to L24 in `main`. I missed that `execute` already prints errors. In that case I'd suggest to change execute to simply `return cmd.Run()`. To me that makes it a little clearer when something is printed.

Reply:
We don't need the additional err check in `execute` if you keep this one here. If you change it to `return cmd.Run()` it will still first run the command and check the error here instead.

Reply:
Thanks @0xFelix - i've made the update.